pom.xml

```diff
@@ -372,7 +372,6 @@
 					<reportSet>
 						<reports>
 							<report>scm</report>
-							<report>dependencies</report>
```

Comment:
Why do you want to remove [that page](https://logging.apache.org/log4cxx/next_stable/dependencies.html)? It currently somewhat documents that `log4j` is necessary for the tests, doesn't look too wrong to me. It would like to keep it for now.

Reply:
I was trying to add the compile time dependencies (APR, APR-Util) and the testcase dependencies (sed, zip, gzip) to the report, but could not because the maven expects all entries in dependencies to be jar files.

I can only see that the dependencies can be mention elsewhere in documentation (hence the the building page change).

Reply:
I get your point and tried the same in the past and failed as well. Tried again now and came up with some hacky approach:

	<dependency>
		<groupId>apr</groupId>
		<artifactId>util</artifactId>
		<version>1.5.4</version>
		<scope>system</scope>
		<systemPath>${user.home}/Documents/Svn/Src/Libs/trunk/C/X-OS/APR/apr-util/1.5.4/build/RAD 10.2/libs/libapr-util/Win32/Debug/out/libapr-util.lib</systemPath>
		<type>lib</type>
	</dependency>

That makes the build succeed and adds dependencies to the page, but still prints an error message and would be best to avoid any hard-coded path. I've [asked on SO](https://stackoverflow.com/questions/62995298/how-to-model-non-java-dependencies-not-available-in-any-repository-using-maven) about that problem, maybe there's some solution with keeping the plugin we are just not aware of.

Reply:
According to comments on the SO-question I created the following result as an example:

![Clipboard01](https://user-images.githubusercontent.com/6223655/88031086-64edbe00-cb3c-11ea-8f17-479c7d856053.png)

The problem is to find the binaries, I didn't find something suitable for ZIP and GZIP yet. GREP is only used as an additional example because it was easily available. In theory we could create our own `pom.xml` describing the individual missing dependencies and maintain those with the project. OTOH this might not be easier instead of simply writing text like you have done already, but that should be a bit more prominent than. I like the scope-stuff already provided by Maven, though.

Thoughts?